salesagility/SuiteCRM
# Update composer.lock + codecov.yml + rebuild SASS/JS

## Description



- Updated composer.lock.
- Changed codecov to error if the ci failed or upload is not found.
- Rebuilt minified SASS/JS files (In this case only JS as there was no SASS rebuild necessary).

## How To Test This

No real way to test except to pull down this PR and rebuild SASS/JS to check I haven't missed any. The composer.lock changes will get tested in the branch.

## Types of changes


### Final checklist
